.github/workflows/ci.yml

```diff
@@ -54,9 +54,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           eslint: true
-          eslint_args: --ignore-pattern test --ignore-pattern vendor
```

Comment:
Allowing eslint in CI to use the same .eslintignore and .eslintrc.js config files